net-misc/networkmanager-fortisslvpn/networkmanager-fortisslvpn-1.2.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+GNOME_ORG_MODULE="NetworkManager-${PN##*-}"
+
+inherit gnome2 autotools
+
+DESCRIPTION="NetworkManager Fortinet SSLVPN compatible plugin"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+
+RDEPEND="
+	>=net-misc/networkmanager-1.2.2:=
+	>=dev-libs/dbus-glib-0.74
+	>=dev-libs/glib-2.32:2
+	>=net-misc/openfortivpn-1.2.0
+	gtk? (
+		app-crypt/libsecret
```

Comment:
You need libsecret 0.18 per https://git.gnome.org/browse/network-manager-fortisslvpn/tree/configure.ac#n101